### PR TITLE
Provide build script

### PR DESCRIPTION
**Pull request by yeesian**

### CODE REVIEW

**Comment by yeesian** [comment]
@tkelman, I haven't been able ot make it work with WinRPM :/ haha

**Comment by tkelman** [comment]
Looks like they only provide the binary as an installer, and it's in some format that 7zip isn't able to easily extract as an archive. There's an example of imagemagick which uses an installer on windows - https://github.com/timholy/Images.jl/blob/master/deps/build.jl

Or if this can be cross-compiled, you could try adapting a spec file for it on opensuse build service. There's an example that uses cmake here https://build.opensuse.org/package/view_file/windows:mingw:win64/mingw64-metis/mingw64-metis.spec?expand=1

**Comment by yeesian** [comment]
> Or if this can be cross-compiled, you could try adapting a spec file for it on opensuse build service. There's an example that uses cmake here https://build.opensuse.org/package/view_file/windows:mingw:win64/mingw64-metis/mingw64-metis.spec?expand=1

It seems possible, from their build & install instructions. If you're around this coming week, it'll be great if you can mentor me through the steps.

**Comment by tkelman** [comment]
Yeah, definitely can do that. There area few manual steps to hook up the opensuse build service repositories properly that I need to write down, will be good to set up from scratch for documentation's sake.

**Comment by yeesian** [comment]
@tkelman I'm getting

**(TravisCI)**
```bash
INFO: Building LibGEOS
INFO: Computing test dependencies for LibGEOS...
INFO: Cloning cache of FactCheck from git://github.com/JuliaLang/FactCheck.jl.git
INFO: Installing FactCheck v0.2.8
INFO: Testing LibGEOS
ERROR: ccall: could not find function GEOSDelaunayTriangulation in library /usr/lib/libgeos_c.so
 in delaunayTriangulation at /home/travis/.julia/v0.3/LibGEOS/src/geos_functions.jl:556
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354
while loading /home/travis/.julia/v0.3/LibGEOS/test/test_geos_functions.jl, in expression starting on line 150
while loading /home/travis/.julia/v0.3/LibGEOS/test/runtests.jl, in expression starting on line 1
```

and 

**(AppVeyor)**
```
Build started
git clone -q git://github.com/JuliaGeo/LibGEOS.jl.git C:\projects\libgeos-jl
git fetch -q origin +refs/pull/7/merge:
git checkout -qf FETCH_HEAD
Running Install scripts
(new-object net.webclient).DownloadFile( $("http://s3.amazonaws.com/"+$env:JULIAVERSION), "C:\projects\julia-binary.exe")
C:\projects\julia-binary.exe /S /D=C:\projects\julia
IF EXIST .git\shallow (git fetch --unshallow)
C:\projects\julia\bin\julia -e "versioninfo(); Pkg.clone(pwd(), \"LibGEOS\"); Pkg.build(\"LibGEOS\")"
Julia Version 0.3.10 
Commit c8ceeef* (2015-06-24 13:54 UTC) 
Platform Info: 
  System: Windows (i686-w64-mingw32) 
  CPU: Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz 
  WORD_SIZE: 32 
  BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Nehalem) 
  LAPACK: libopenblas 
  LIBM: libopenlibm 
  LLVM: libLLVM-3.3 
INFO: Initializing package repository C:\Users\appveyor\.julia\v0.3
INFO: Cloning METADATA from git://github.com/JuliaLang/METADATA.jl
INFO: Cloning LibGEOS from C:\projects\libgeos-jl
Note: checking out '094418262b7af2454c6fe81ceb23a358b2319b95'.
 
You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.
 
If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:
 
  git checkout -b new_branch_name
 
INFO: Computing changes...
INFO: Cloning cache of BinDeps from git://github.com/JuliaLang/BinDeps.jl.git
INFO: Cloning cache of Compat from git://github.com/JuliaLang/Compat.jl.git
INFO: Cloning cache of GeoInterface from git://github.com/JuliaGeo/GeoInterface.jl.git
INFO: Cloning cache of LibExpat from git://github.com/amitmurthy/LibExpat.jl.git
INFO: Cloning cache of SHA from git://github.com/staticfloat/SHA.jl.git
INFO: Cloning cache of URIParser from git://github.com/JuliaWeb/URIParser.jl.git
INFO: Cloning cache of WinRPM from git://github.com/JuliaLang/WinRPM.jl.git
INFO: Cloning cache of Zlib from git://github.com/dcjones/Zlib.jl.git
INFO: Installing BinDeps v0.3.12
INFO: Installing Compat v0.4.6
INFO: Installing GeoInterface v0.0.1
INFO: Installing LibExpat v0.0.8
INFO: Installing SHA v0.0.4
INFO: Installing URIParser v0.0.5
INFO: Installing WinRPM v0.1.9
INFO: Installing Zlib v0.1.8
INFO: Building WinRPM
WARNING: skipping repodata/repomd.xml, not in cache -- call WinRPM.update() to download
WARNING: skipping repodata/repomd.xml, not in cache -- call WinRPM.update() to download
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/e22c34ef1521489bb0d9919f340a28cf3af47e97d76d55744f0f79f4ccaa9653-primary.xml.gz
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/b7cb653ff6c7726491474dea31a8153f73907e6887106f8b2d6bb1c93208023e-primary.xml.gz
INFO: Package database updated
INFO: Building WinRPM
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Building LibGEOS
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/openSUSE_13.1/repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/openSUSE_13.1/repodata/c1f17fdba2b21f907d2d73615270696987d53bce56644c04099eb595d29132b9-primary.xml.gz
INFO: Updating WinRPM package list
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/openSUSE_13.1/repodata/repomd.xml
===============================[ ERROR: LibGEOS ]===============================
 
None of the selected providers can install dependency libgeos.
Use BinDeps.debug(package_name) to see available providers
 
while loading C:\Users\appveyor\.julia\v0.3\LibGEOS\deps\build.jl, in expression starting on line 26
 
================================================================================

**Comment by tkelman** [comment]
for 32 bit windows we need to make a new package, an exact copy of the win64 version but replace mingw64 with mingw32

**Comment by yeesian** [comment]
`%{_mingw32_bindir}/x86_64-w64-mingw32-geos-config` remains unchanged?

**Comment by tkelman** [comment]
Probably need to replace `x86_64` with `i686`

**Comment by yeesian** [comment]
should i copy from https://build.opensuse.org/package/show/home:rogeroberholtzer:mingw/mingw32-libgeos? haha, it's pretty depressing how most of the rest don't work at all

**Comment by tkelman** [comment]
Looking at his spec file, it looks very very similar to what we came up with in an hour today. The `objdump` macros at the top aren't needed any more, the way we have it is cleaner and newer.

**Comment by yeesian** [comment]
yeah, I've been looking at the diff. I tried both his and yours, and it's reporting unresolvable for both.

**Comment by tkelman** [comment]
oh, you need to do the add repositories thing but for windows:mingw32 this time

**Comment by yeesian** [comment]
It builds successfully now! (:

Do I just push the new url to WinRPM's sources as well?

**Comment by yeesian** [comment]
Yay, it works now! But I think my `build.jl` file isn't getting it right.

```
INFO: Building WinRPM
WARNING: skipping repodata/repomd.xml, not in cache -- call WinRPM.update() to download
WARNING: skipping repodata/repomd.xml, not in cache -- call WinRPM.update() to download
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/e22c34ef1521489bb0d9919f340a28cf3af47e97d76d55744f0f79f4ccaa9653-primary.xml.gz
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/b7cb653ff6c7726491474dea31a8153f73907e6887106f8b2d6bb1c93208023e-primary.xml.gz
INFO: Package database updated
INFO: Building WinRPM
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Building LibGEOS
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/openSUSE_13.1/repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/openSUSE_13.1/repodata/c1f17fdba2b21f907d2d73615270696987d53bce56644c04099eb595d29132b9-primary.xml.gz
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/windows_mingw_win32_openSUSE_Factory/repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/windows_mingw_win32_openSUSE_Factory/repodata/42aa036fa2a246da177827d84cfee865197733d181ba516e5ca492b195e0dde9-primary.xml.gz
INFO: Updating WinRPM package list
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win32/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/windows:/mingw:/win64/openSUSE_13.1//repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/openSUSE_13.1/repodata/repomd.xml
INFO: Downloading http://download.opensuse.org/repositories/home:yeesian/windows_mingw_win32_openSUSE_Factory/repodata/repomd.xml
INFO: Packages to install: libgeos-devel, libstdc++6, libgcc_s_sjlj1, libgeos
INFO: Downloading: libgeos-devel
INFO: Extracting: libgeos-devel
INFO: Downloading: libstdc++6
INFO: Extracting: libstdc++6
INFO: Downloading: libgcc_s_sjlj1
INFO: Extracting: libgcc_s_sjlj1
INFO: Downloading: libgeos
INFO: Extracting: libgeos
INFO: Complete
===============================[ ERROR: LibGEOS ]===============================
 
Provider PackageManager failed to satisfy dependency libgeos
while loading C:\Users\appveyor\.julia\v0.3\LibGEOS\deps\build.jl, in expression starting on line 27
 
================================================================================
 
================================[ BUILD ERRORS ]================================
 
WARNING: LibGEOS had build errors.
 
 - packages with build errors remain installed in C:\Users\appveyor\.julia\v0.3
 - build the package(s) and all dependencies with `Pkg.build("LibGEOS")`
 - build a single package by running its `deps/build.jl` script
 
================================================================================
```

**Comment by tkelman** [comment]
Try adding another alias for `libgeos_c-1`

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    - linux
+julia:
+    - release
+    - nightly
+notifications:
+    email: false
+script:
+    - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+    - julia -e 'Pkg.clone(pwd()); Pkg.build("LibGEOS"); Pkg.test("LibGEOS"; coverage=true)';
+    - julia -e 'cd(Pkg.dir("LibGEOS")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```
Comment (IainNZ) [inline .travis.yml]: You should put this as after_success in case Coverage breaks

Comment (yeesian) [inline .travis.yml]: This was copied from Example.jl. I'll submit a PR there too

**Comment by yeesian** [comment]
@garborg I think this branch is ready for your review (after AppVeyor turns green). I suspect the errors from travis are due to features available in geos 3.4.* that aren't available in earlier versions.

**Comment by garborg** [comment]
Tried to take it for a spin locally, but 0.4 is a little broken right now, so I'll revisit. @yeesian, not sure if there's anything to do about the library version on Travis.

**Comment by yeesian** [comment]
From what I understand, [ubuntu isn't on geos 3.4.2 yet](http://packages.ubuntu.com/hu/source/precise/geos). I don't know enough to help push things forward there though.

**Comment by tkelman** [comment]
When you were getting `ERROR: ccall: could not find function GEOSDelaunayTriangulation in library /usr/lib/libgeos_c.so` was that with a source build? You could try commenting AptGet out for now and see how much work it takes to get the source build working again

**Comment by yeesian** [comment]
Ah yeah, Travis is failing on the same line. I'm confused, haha

**Comment by tkelman** [comment]
are there optional libraries libgeos depends on, where it builds delaunay functionality when they're present but skips some features when the dependencies aren't available?

**Comment by yeesian** [comment]
I've dug around [here](https://github.com/libgeos/libgeos/blob/svn-trunk/README) and [here](https://trac.osgeo.org/geos/wiki/BuildingOnUnixWithAutotools), but can't find anything.

**Comment by staticfloat** [comment]
I can produce this, or a very similar failure, locally using a vagrant ubuntu machine:

```
ERROR: ccall: could not find function initGEOS in library /home/vagrant/.julia/v0.3/LibGEOS/deps/usr/lib/libgeos.so
 in GEOSconnection at /home/vagrant/.julia/v0.3/LibGEOS/src/LibGEOS.jl:32
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in reload_path at loading.jl:152
 in _require at loading.jl:67
 in require at loading.jl:51
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354
```

Dumping the symbols defined in `libgeos.so` shows that there is indeed no `initGEOS` in it.  In fact, that symbol is in `libgeos_c.so`.  Do we actually want to load `libgeos.so` at all?

**Comment by yeesian** [comment]
Oh! We probably want libgeos_c.so instead, I think. I'm running and testing everything off OS X, so it was mostly guesswork on my part. Would it work with `libgeos_c.so`?

**Comment by staticfloat** [comment]
Wait, so which files the symbols are in depends on what platform it's compiled on?  That's no fun!

**Comment by yeesian** [comment]
Ohwait, the homebrew setup didnt work either. It should probably be `libgeos_c` on all platforms?

**Comment by staticfloat** [comment]
Try changing the beginning of your build script to this:
```
libgeos = library_dependency("libgeos",aliases=["libgeos_c", "libgeos_c-1"], validate = function(path, handle)
    return dlsym_e(handle,:initGEOS) != C_NULL
end)
```

This sets up a validation function that will only return true on the library that actually has `initGEOS` inside of it.  Therefore, regardless of whether it's `libgeos.so` or `libgeos_c.so`, it'll try both and only load the one with the actual symbol inside of it.  With this modification, my local Ubuntu 12.04 passes the tests.

**Comment by yeesian** [comment]
Yup, I can verify the Homebrew formula works for me now, albeit with the following warnings when I build it:

```julia
  | | |_| | | | (_| |  |  Version 0.4.0-pre+7105 (2015-08-30 15:16 UTC)
 _/ |\__'_|_|_|\__'_|  |  Commit 349a4e1 (2 days old master)
|__/                   |  x86_64-apple-darwin13.4.0

julia> using LibGEOS
ERROR: LoadError: ccall: could not find function initGEOS in library /usr/local/lib/libgeos.dylib
 in call at /Users/yeesian/.julia/v0.4/LibGEOS/src/LibGEOS.jl:32
 in include at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in include_from_node1 at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in require at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
while loading /Users/yeesian/.julia/v0.4/LibGEOS/src/LibGEOS.jl, in expression starting on line 40

julia> Pkg.build("LibGEOS")
INFO: Building Homebrew
HEAD is now at 4294a6e cheat: update 2.1.14 bottle.
HEAD is now at c490b44 Merge pull request #61 from staticfloat/staging
INFO: Building LibGEOS
WARNING: dlsym_e is deprecated, use Libdl.dlsym_e instead.
 in depwarn at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in dlsym_e at deprecated.jl:30
 in anonymous at /Users/yeesian/.julia/v0.4/LibGEOS/deps/build.jl:5
 in check_system_handle! at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:540
 in check_path! at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:513
 in _find_library at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:498
 in satisfy! at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:718 (repeats 2 times)
 in anonymous at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:773
 in include at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in include_from_node1 at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in evalfile at loading.jl:287 (repeats 2 times)
 in anonymous at pkg/entry.jl:659
 in cd at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in build! at pkg/entry.jl:658
 in build at pkg/entry.jl:670
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in build at pkg.jl:64
while loading /Users/yeesian/.julia/v0.4/LibGEOS/deps/build.jl, in expression starting on line 32
WARNING: dlsym_e is deprecated, use Libdl.dlsym_e instead.
 in depwarn at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in dlsym_e at deprecated.jl:30
 in anonymous at /Users/yeesian/.julia/v0.4/LibGEOS/deps/build.jl:5
 in check_system_handle! at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:540
 in check_path! at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:513
 in _find_library at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:498
 in satisfy! at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:718 (repeats 2 times)
 in anonymous at /Users/yeesian/.julia/v0.4/BinDeps/src/dependencies.jl:773
 in include at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in include_from_node1 at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in evalfile at loading.jl:287 (repeats 2 times)
 in anonymous at pkg/entry.jl:659
 in cd at /Applications/Julia-0.4.0-pre-349a4e1977.app/Contents/Resources/julia/lib/julia/sys.dylib
 in build! at pkg/entry.jl:658
 in build at pkg/entry.jl:670
 in anonymous at pkg/dir.jl:31
 in cd at file.jl:22
 in cd at pkg/dir.jl:31
 in build at pkg.jl:64
while loading /Users/yeesian/.julia/v0.4/LibGEOS/deps/build.jl, in expression starting on line 32
```

**Comment by staticfloat** [comment]
Ah, yeah, I was testing on 0.3.  You'd best use `Compat` to get `Libdl.dlsym_e` on `v0.3`.

**Comment by yeesian** [comment]
It works for mac, but not linux, on travis now. May I check what version of libgeos it fetches on ubuntu for you?

**Comment by staticfloat** [comment]
For some reason, on Travis, [it's finding a `libgeos_c.so` in `/usr/lib`](https://travis-ci.org/JuliaGeo/LibGEOS.jl/jobs/78344995#L146).  Can you try commenting out the `provides(AptGet,"libgeos-dev", libgeos)` line?

**Comment by yeesian** [comment]
hm, same thing

**Comment by staticfloat** [comment]
I am very confused as to why we are apparently finding `libgeos_c.so` in `/usr/lib`:

```
ERROR: ccall: could not find function GEOSDelaunayTriangulation in library /usr/lib/libgeos_c.so
```

My best guess is that for whatever reason, an old version of `libgeos_c.so` is actually preloaded on to the Travis machines!  In which case, let's add the `GEOSDelaunayTriangulation` symbol to our verification function:


```
validate = function(path, handle)
+    return @compat(Libdl.dlsym_e(handle,:initGEOS)) != C_NULL && @compat(Libdl.dlsym_e(handle,:GEOSDelaunayTriangulation)) != C_NULL
+end
```

**Comment by staticfloat** [comment]
Woohoo! *high-fives*

**Comment by yeesian** [comment]
We're seeing green, thanks! Do the lines

```julia
provides(Sources, URI("http://download.osgeo.org/geos/geos-$(version).tar.bz2"), [libgeos], os = :Unix)
```

and beyond get run only if the libraries that are preloaded onto Travis fails the `validate` function?

**Comment by staticfloat** [comment]
Yes. BinDeps has built-in priorities, so libraries that are already on a
computer will always trump installing new binaries, unless you reject the
current binaries somehow.

**Comment by yeesian** [comment]
@garborg good to merge?

**Comment by garborg** [comment]
@yeesian It looks like the right direction to me -- I say go ahead. Won't be able to take it for a spin today, but hopefully later in the week.

**Comment by yeesian** [comment]
A'ight, I'm going for it then! (:

Feel free to file issues/etc

**Comment by tkelman** [comment]
@yeesian it looks like you'll need to go into the opensuse build service and enable your repository to build on 13.2 and change the repo here accordingly

**Comment by yeesian** [comment]
Hm, it's enabled, but unresolvable with
```
nothing provides mingw32-cross-binutils, nothing provides mingw32-cross-gcc, nothing provides mingw32-cross-gcc-c++, nothing provides mingw32-cross-pkg-config, nothing provides mingw32-filesystem
```

**Comment by tkelman** [comment]
Just needs some time to build the world, see https://build.opensuse.org/project/monitor/windows:mingw:win64

**Comment by yeesian** [comment]
What needs to change in this repo? Remind me I should set up travis and appveyor badges

**Comment by tkelman** [comment]
https://github.com/JuliaGeo/LibGEOS.jl/blob/f882a4b17d3995a599d3f8c12ceb2a318c758716/deps/build.jl#L18-L19 should change from 13.1 to 13.2, I think that's it.

**Pull request merged**